src/lib.rs

```diff
@@ -165,18 +169,21 @@ struct State {
     input_state: InputState,
     egui_state: egui_winit::State,
     context: egui::Context,
-    rpass: RenderPass,
+    egui_renderer: egui_wgpu::Renderer,
     import_error: String,
     #[cfg(not(target_arch = "wasm32"))]
     clipboard: arboard::Clipboard,
 }
 
 impl State {
-    async fn new(window: &Window, ev_loop: &EventLoop<()>) -> Self {
+    async fn new(window: &Window, event_loop: &EventLoop<()>) -> Self {
         let size = window.inner_size();
 
-        let instance = wgpu::Instance::new(wgpu::Backends::all());
-        let surface = unsafe { instance.create_surface(window) };
+        let instance = wgpu::Instance::new(wgpu::InstanceDescriptor {
```

Comment:
you can just use `wgpu::Instance::new(Default::default())` here (which is what I've done in the latest commits)